src/main/webapp/WEB-INF/static/js/trialmanager/germplasm-study-source/germplasm-study-source-tab.html

```diff
@@ -15,6 +15,12 @@
 					<label>Selected:</label>
 					<span class="control-label-bold">{{isAllPagesSelected ? getRecordsFiltered() : size(selectedItems)}}</span>
 				</div>
+				<div class="form-group  checkbox-inline">
+					<label style="margin-left: 15px">
+						<input type="checkbox" ng-checked="isAllPagesSelected" ng-click="onSelectAllPages()" id="check-select-all">
```

Comment:
Maybe it's a good idea to hide the checkbox column if all page checkbox is selected.